devices/legrand.js

```diff
@@ -28,6 +28,22 @@ const tzLocal = {
 };
 
 module.exports = [
+    {
+        zigbeeModel: [' Pocket remote\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000'+
+            '\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000\u0000'],
+        model: '067755',
+        vendor: 'Legrand',
+        description: 'Wireless and batteryless 4 scenes control',
+        fromZigbee: [fz.identify, fz.command_on, fz.command_off, fz.command_toggle, fz.battery, fz.command_recall],
+        exposes: [e.battery(), e.action(['identify', 'recall_1_1'])],
```

Comment:
Does it only send a `recall_1_1` action? I would expect more, otherwise `fz.command_on, fz.command_off, fz.command_toggle` can be removed.

Reply:
You're right, I just did it. (I confirm that I've got only recall_1_1)